app/models/compute_resources/foreman/model/vmware.rb

```diff
@@ -366,7 +378,8 @@ def clone_vm(raw_args)
         "numCPUs" => args[:cpus],
         "memoryMB" => args[:memory_mb],
         "datastore" => args[:volumes].first[:datastore],
-        "resource_pool" => [args[:cluster], args[:resource_pool]],
+        "storage_pod" => args[:volumes].first[:storage_pod],
+        #"resource_pool" => [args[:cluster], args[:resource_pool]],
```

Comment:
I could not reproduce this error anymore. I was probably just forgetting something...